src/test/java/htsjdk/samtools/util/BufferedLineReaderTest.java

```diff
@@ -71,10 +71,12 @@ private void fromStringEmptyTestHelper(final String terminator, final boolean la
             input = "";
         }
         final BufferedLineReader blr = BufferedLineReader.fromString(input);
+        blr.peek();
```

Comment:
Instead of adding these peek calls in the existing tests, would it be possible to add a new test that includes a file with windows line endings?  Ideally a test would fail on all machines without your change, instead of just on windows ones.

Reply:
I was pleasantly surprised to see that the test was already explicitly testing different terminators (see TERMINATORS), it was just not exercising peek(). That is, the test does fail on non-windows machines when the mark size is 1 (I dev on linux). So an external file seems redundant, right?

Reply:
Hmn, that's a good point.  I don't think I was paying that much attention to what was being tested exactly.